packages/config/src/lib/createPlugins.ts

```diff
@@ -7,9 +10,40 @@ const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin'
 const { glob } = require('glob');
 const path = require('path');
 
-module.exports = ({ rootFolder, insights, generateSourceMaps, plugins, definePlugin = {}, _unstableHotReload = false, useFileHash = true } = {}) => {
+export type WebpackPluginDefinition = undefined | null | false | '' | 0 | ((this: Compiler, compiler: Compiler) => void) | WebpackPluginInstance;
+
+export type CreatePluginsOptions = {
+  rootFolder: string;
```

Comment:
NIT - If some of these keys (such as `rootFolder`) are present and the same type between `CreateConfigOptions` and `CreatePluginsOptions` I wonder if we move those to a common type that can be extended to avoid duplication?

Reply:
I think we should.